cookbook/mass_edition/dont_apply_rules_on_mass_edit_action.rst

```diff
@@ -0,0 +1,113 @@
+How to Avoid Rules Execution on Mass Edit Actions
+=================================================
+
+| The Akeneo PIM comes with a number of mass edit actions.
+| By default in Enterprise Edition, after each of those actions, **rules are recalculated** for edited products.
```

Comment:
**rules are applied** for edited products in enterprise edition by default  -> I find 'applied' more accurate as rules are applied but their operations calculates new values ? ahah :)